cl/opinion_page/views.py

```diff
@@ -305,6 +328,18 @@ def view_opinion(request, pk, _):
         'private': cluster.blocked,
         'citing_clusters': citing_clusters,
         'top_authorities': cluster.authorities[:5],
+        'related_algorithm': 'mlt',
```

Comment:
Is it crazy to put all of this into one variable to be handled in the template. This is a lot of variables to be passing to the templates.

Reply:
We need to pass all other parameters to have consistent search filters. I've refactored this to get the statuses from `models.DOCUMENT_STATUSES`.